PowerShell.Common.props

```diff
@@ -1,5 +1,9 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+    <PropertyGroup>
+      <LangVersion>Latest</LangVersion>
```

Comment:
Why not put this tag in the `<PropertyGroup>` at the end of this file?

Reply:
Done.